libvast/builtins/operators/inspect.cpp

```diff
@@ -0,0 +1,154 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2023 The VAST Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include "vast/pipeline.hpp"
```

Comment:
Nitpick: `#include <vast/...>`

Reply:
I think we all just copy `pass.cpp` every time we create a new operator, or so it feels... 😅